ccd-definition/AuthorisationCaseField/CareSupervision/solicitor.json

```diff
@@ -3,7 +3,7 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseFieldID": "caseHistory",
-    "UserRole": "[SOLICITOR]",
+    "UserRole": ["[SOLICITOR]"],
```

Comment:
UserRole -> UserRoles

Reply:
Done. Missed a commit push but should be there now.